projects/common/lib/components/button/button.component.ts

```diff
@@ -25,9 +25,9 @@ import { ControlContainer, ControlValueAccessor, NgForm, NG_VALUE_ACCESSOR } fro
 })
 
 export class ButtonComponent implements OnInit {
-  
-  // Can pass the Style class of a button e.g. For primary, btn btn-primary. Default, btn btn-default. Error, btn btn-danger 
-  @Input() styleClass: string = 'btn btn-default'; 
+
+  // Can pass the Style class of a button e.g. For primary, btn btn-primary. Default, btn btn-default. Error, btn btn-danger
+  @Input() styleClass: string = 'btn btn-default';
```

Comment:
We should _always_ include the "btn" class, so take it out from this string and hardcode it into the HTML (e.g. `class="btn btn-{{styleClass}}`).  That way when using it you'd only have to do `styleClass='default'` or `styleClass='primary`.

In fact we should take it one step further, rename it to 'buttonType' and lock down the valid parameters:

```suggestion
  @Input() buttonType: 'default' | 'primary' | 'secondary' = 'default'
```

This locks down the CSS and valid button types. Now a user can't add in like "btn-success" or something that breaks our designs and we don't want.